web/src/App.tsx

```diff
@@ -32,19 +36,31 @@ const App = (): ReactElement => {
     setJpalTrackingCode(safeLocalStorage.getItem(JPAL_TRACKING_CODE_KEY))
   }, [])
 
+  // TODO upgrade mui
+  // There are some errors in the console due to mui
+  // https://github.com/mui/material-ui/issues/45432
   return (
     <ThemeContainer contentDirection={contentDirection}>
-      <I18nProvider contentLanguage={contentLanguage}>
-        <>
-          <Helmet pageTitle={t('pageTitle')} rootPage />
-          <Router>
-            <GlobalStyle />
-            <TtsContainer languageCode={contentLanguage}>
-              <RootSwitcher setContentLanguage={setContentLanguage} />
-            </TtsContainer>
-          </Router>
-        </>
-      </I18nProvider>
+      <ThemeProvider
+        theme={createTheme({
+          colorSchemes: {
+            light: buildConfig().lightTheme,
+            dark: buildConfig().darkTheme,
+          },
+        })}>
+        <Button startIcon={<RocketLaunch color='secondary' />}>Test</Button>
```

Comment:
Do we need that?

Reply:
We do not need this, its more a proof of concept that mui works as expected. I'll remove it later.